jestconfig.json

```diff
@@ -3,6 +3,9 @@
     "raf/polyfill",
     "./jestsetup.js"
   ],
+  "testMatch": [
```

Comment:
Curious, we didn't update any of the jest tests, so why did we add this?

Reply:
Jest matching is pretty generous and was trying to match on our .test.jsx examples for terra-dev-site. this restricts jest matching to look for jest tests only in the the jest folder

Reply:
I think I must have asked this before because that response sounds similar.